app/mattermost_deprecated.js

```diff
@@ -0,0 +1,660 @@
+// Copyright (c) 2016-present Mattermost, Inc. All Rights Reserved.
```

Comment:
Can we just remove this since it's not used any more, or is there a reason for keeping it around?

Reply:
Nope, no reason.  I'll remove